content/e/kpw/2016-11-26-kpw-arena-4.md

```diff
@@ -31,11 +31,11 @@ Nowy Rozdział (_The New Chapter_) was held in Gdańsk, in the gymnastics hall o
   - '[Elliott Jordan](@/w/elliott-jordan.md)'
   - c: '[KPW Championship](@/c/kpw-championship.md)'
 - credits:
-    Ring announcer: '???'
-    Referees: '[Krystian Malinowski](@/w/krystian-malinowski.md), ???'
+    Ring announcer: 'Karolina Wensierska'
+    Referees: '[Krystian Malinowski](@/w/krystian-malinowski.md), [Karol Górski](@/w/iskra.md)'
 {% end %}
 
-This would be [Damian Lambert's](@/w/damien-rothschild.md) last appearance for KPW.
+This would be [Damian Lambert's](@/w/damien-rothschild.md) last appearance for KPW. As [Arkadiusz Pawłowski](@/w/pan-pawlowski.md) was unavailable that day, his duties were taken over by [Kaszub's](@/w/kaszub.md) sister, Karol Wensierska.
```

Comment:
```suggestion
This would be [Damian Lambert's](@/w/damien-rothschild.md) last appearance for KPW. As [Arkadiusz Pawłowski](@/w/pan-pawlowski.md) was unavailable that day, his duties were taken over by [Kaszub's](@/w/kaszub.md) real-life sister, Karolina Wensierska.
```